cmd/podman/generate/systemd.go

```diff
@@ -81,6 +81,10 @@ func init() {
 	flags.BoolVar(&systemdOptions.New, newFlagName, false, "Create a new container or pod instead of starting an existing one")
 	flags.BoolVarP(&systemdOptions.NoHeader, "no-header", "", false, "Skip header generation")
 
+	headerFlagName := "header"
+	flags.StringVar(&systemdOptions.Header, headerFlagName, "", "Append the specified header.")
```

Comment:
Mini nit: no trailing dot for consistencty
```suggestion
	flags.StringVar(&systemdOptions.Header, headerFlagName, "", "Append the specified header")
```